File: model/src/main/java/com/sealinkin/oset/model/Oset_ShipperLineModel.java
package com.sealinkin.oset.model;

import com.sealinkin.oset.po.Oset_ShipperLine;

public class Oset_ShipperLineModel extends Oset_ShipperLine{

	private static final long serialVersionUID = 1L;
	
	private String enterpriseNo;
	private String warehouseNo;
	private String shipperNo;
	private String lineNo;
	
	private String shipperName;
	private String lineName;
	public String getWarehouseNo() {
		return warehouseNo;
	}
	public void setWarehouseNo(String warehouseNo) {
		this.warehouseNo = warehouseNo;
	}
	public String getShipperNo() {
		return shipperNo;
	}
	public void setShipperNo(String shipperNo) {
		this.shipperNo = shipperNo;
	}
	public String getLineNo() {
		return lineNo;
	}
	public void setLineNo(String lineNo) {
		this.lineNo = lineNo;
	}
	public String getShipperName() {
		return shipperName;
	}
	public void setShipperName(String shipperName) {
		this.shipperName = shipperName;
	}
	public String getLineName() {
		return lineName;
	}
	public void setLineName(String lineName) {
		this.lineName = lineName;
	}
	public String getEnterpriseNo() {
		return enterpriseNo;
	}
	public void setEnterpriseNo(String enterpriseNo) {
		this.enterpriseNo = enterpriseNo;
	}
	
}

File: wms/src/main/webapp/system/app/model/oset/oset_ShipperLineModel.js
 Ext.define('cms.model.oset.oset_ShipperLineModel', {
    extend: 'Ext.data.Model',
    fields: [
    	{name:'warehouseNo'},
    	{name:'shipperNo'},
    	{name:'lineNo'},
    	{name:'rgstName'},
    	{name:'rgstDate'},
    	{name:'updtName'},
    	{name:'updtDate'},
    	
    	{name:'shipperName'},
    	{name:'lineName'}
    ],
    idProperty:'warehouseNo,shipperNo,lineNo'
 });